examples/compute/cuda/hello_compute.cu

```diff
@@ -60,7 +60,8 @@ int hpx_main(boost::program_options::variables_map& vm)
             int i = blockDim.x * blockIdx.x + threadIdx.x;
             if(i < N)
                 C[i] = A[i] + B[i];
-        }, d_A.data(), d_B.data(), d_C.data());
+        }, (int*)(d_A.data()), (int*)(d_B.data()), (int*)(d_C.data())
+        );
 
```

Comment:
But this cast is clearly wrong as it happens in host code where `d_A.data()` returns a `target_ptr<int>` and not an `int*`. I rather think this is a bug and should be reported to the clang guys.